AWS-VPC_deploy.yaml

```diff
@@ -0,0 +1,113 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: VPC with 2 public subnets and 2 private subnets
+
+Resources:
+  VPC:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: 10.0.0.0/16
+      EnableDnsSupport: true
+      EnableDnsHostnames: true
+      Tags:
+        - Key: Name
+          Value: AWS-VPC
+
+  InternetGateway:
+    Type: AWS::EC2::InternetGateway
+    Properties:
+      Tags:
+        - Key: Name
+          Value: MyIGW
+
+  VPCGatewayAttachment:
```

Comment:
Rename it in a way that shows the gateway type. Example, `InternetGatewayAttachment`

Reply:
renamed